src/typography/expandable-text.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ExpandableText = ({ title, children }) => (
```

Comment:
I think I added the [`transform-object-rest-spread`](https://babeljs.io/docs/plugins/transform-object-rest-spread/) in a previous PR @lennym could you just validate this all works correctly in the bundled components?